app/src/main/java/woowacourse/movie/domain/Price.kt

```diff
@@ -0,0 +1,9 @@
+package woowacourse.movie.domain
+
+import java.io.Serializable
+
+class Price(val value: Int = DEFAULT_PRICE) : Serializable {
+    companion object {
+        private const val DEFAULT_PRICE = 13000
```

Comment:
가독성을 위해 아래처럼 표기하는것도 좋아요!
```suggestion
        private const val DEFAULT_PRICE = 13_000
```